TestApp/Model/Identifier.cs

```diff
@@ -0,0 +1,36 @@
+﻿using FluentValidation;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TestApp.Model
+{
+    public class Identifier
+    {
+        [Key]
+        public Guid id { get; set; }
+        [NotMapped]
+        public IdentificationType idenityType { get; set; }
```

Comment:
what about identifier type? why is not mapped?

Reply:
There is no need  of mapping because we have already value in this entity which saved to DB, and we can map the value to enum  whenever needed.